drivers/pwm/pwm_mchp_xec.c

```diff
@@ -368,6 +373,50 @@ static int pwm_xec_get_cycles_per_sec(const struct device *dev,
 	return 0;
 }
 
+#ifdef CONFIG_PM_DEVICE
+static int pwm_xec_pm_action(const struct device *dev, enum pm_device_action action)
+{
+	const struct pwm_xec_config *const devcfg = dev->config;
+	struct pwm_regs * const regs = devcfg->regs;
+	struct pwm_xec_data * const data = dev->data;
+	int ret = 0;
+
+	switch (action) {
+	case PM_DEVICE_ACTION_RESUME:
+		ret = pinctrl_apply_state(devcfg->pcfg, PINCTRL_STATE_DEFAULT);
+		if (ret != 0) {
+			LOG_ERR("XEC PWM pinctrl setup failed (%d)", ret);
```

Comment:
Sorry only noticed after hitting merge, should this have a `return ret` here? Also `ret` does not need to be initialized.

Reply:
Ok, I update on the next commit